config/docdb/config.go

```diff
@@ -17,6 +17,9 @@ func Configure(p *config.Provider) {
 			}
 			return conn, nil
 		}
+		r.References["db_cluster_parameter_group_name"] = config.Reference{
+			Type: "DBClusterParameterGroup",
```

Comment:
nit: When defining the cross-resource references, we have a more "robust" way of specifying the reference target type. You may also use the config.Reference.TerraformName instead of config.Reference.Type. We consider it as more robust because the generated type names/groups may change via upjet's resource configuration API, whereas the Terraform name is more stable.

Reply:
Thanks, will update the reference.

Reply:
Thanks for the explanation! I've noticed this preference in some PR comments, but never understood why this method was preferred.